src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -93,6 +96,14 @@ private CloudSdk(Path sdkPath, String appCommandMetricsEnvironment,
     this.processRunner = new DefaultProcessRunner(async, stdOutLineListeners, stdErrLineListeners,
         exitListeners, startListeners);
 
+    // Populate jar locations.
+    // TODO(joaomartins): Consider case where SDK doesn't contain these jars. Only App Engine
+    // SDK does.
+    JAR_LOCATIONS.put("servlet-api.jar",
```

Comment:
It's implemented as a map, but provided as API methods. Do you have any alternative suggestion?